baggers.js

```diff
@@ -15440,6 +15440,92 @@ var data = {
         "twitter": "dedece35",
         "mail": "ddecarvalho@pagesjaunes.fr"
       }
+    },
+    {
+      "since": "2017-06-30",
+      "name": "Yannick DALBIN",
+      "bio": "Développeur passionné par le web depuis tout petit, aujourd'hui co-foundateur d'HashtagTrophy, formateur et co-fondateur de Startup Marseille - premier espace de co-living dédié aux Startups sur Marseille.",
+      "picture": "https://fr.gravatar.com/userimage/27284879/f77fd2b6d51b61605576721be9a8dc47.jpeg?size=200",
+      "websites": [
+        {
+          "name": "Web",
+          "url": "http://hashtagtrophy.com"
+        },
+        {
+          "name": "Web",
+          "url": "https://yannickdalbin.com"
+        },
+        {
+          "name": "Web",
+          "url": "https://startupmarseille.com"
+        },
+        {
+          "name": "Web",
+          "url": "https://youtube.com/yannickdalbin"
+        },
+        {
+          "name": "Web",
+          "url": "https://twitter.com/yannickda"
+        },
+        {
+          "name": "Web",
+          "url": "https://facebook.com/yannickdalbin"
+        },
+        {
+          "name": "LinkedIn",
+          "url": "https://linkedin.com/in/yannickda"
+        },
+        {
+          "name": "GitHub",
+          "url": "https://github.com/yannickda"
+        }
+      ],
+      "location": "Marseille et environs",
+      "sessions": [
+        {
+          "title": "Présentation de React.js",
+          "abstract": "React.js tout le monde en parle et ce n'est pas pour rien ! Cette librairie bouscule les codes de développement pour le frontend et rend le dév passionant en plus d'être très rapide. Je vous propose de faire un tour d'horizon afin d'en savoir plus.",
+          "tags": [
+            "Javascript",
+            "React.js",
+            "Librairie",
+            "Vue",
+            "Component"
+          ],
+          "lang": [
+            "fr"
+          ]
+        },
+        {
+          "title": "ES6",
+          "abstract": "Réaliser un projet Javascript sans ES6 parraît aujourd'hui impensable ! Je vous propose de découvrir les nouveautés du langage et vous allez vite voir que son utilisation est indispensable !",
+          "tags": [
+            "Javascript",
+            "ES6"
+          ],
+          "lang": [
+            "fr"
+          ]
+        },
+        {
+          "title": "Webpack",
+          "abstract": "Souvent considéré comme compliqué à prendre en main, webpack semble tout de même s'imposer comme outils de développement. Je vous propose de faire un tour d'horizon de cet outils magique qui va changer votre façon de gérer les assets",
+          "tags": [
+            "Javascript",
+            "Webpack"
+          ],
+          "lang": [
+            "fr"
+          ]
+        }
+      ],
+      "cities": [
+        "Marseille"
```

Comment:
It's is a little far away for me... In Marseille, we have lot of cars and not enough bus or something like that :)